lib/create_or_edit_brew.dart

```diff
@@ -33,6 +33,7 @@ class CreateOrEditBrewState extends State<CreateOrEditBrew> {
 
   String _roaster = "";
   String _blend = "";
+  String _varietal = "";
```

Comment:
You need to still create the RawAutocomplete in create_or_edit_brew.dart which serves as the input for the varietal.

- Modify the _buildForm method to get the value from args.
- Modify the _buildForm method to add another RawAutocomplete child to the main Column, which serves as the input for the _varietal variable. This is similar to how Roaster and Blend have been done in the code. The RawAutocomplete has the following options: optionsBuilder, initialValue, displayStringForOption, fieldViewBuilder, optionsViewBuilder